pkg/service/did/service.go

```diff
@@ -3,23 +3,26 @@ package did
 import (
 	"context"
 	"fmt"
+	"net/http"
 
 	didsdk "github.com/TBD54566975/ssi-sdk/did"
 	sdkutil "github.com/TBD54566975/ssi-sdk/util"
 	"github.com/pkg/errors"
-
 	"github.com/tbd54566975/ssi-service/config"
 	"github.com/tbd54566975/ssi-service/internal/did"
 	"github.com/tbd54566975/ssi-service/internal/util"
+	"github.com/tbd54566975/ssi-service/pkg/service/did/resolve"
 	"github.com/tbd54566975/ssi-service/pkg/service/framework"
 	"github.com/tbd54566975/ssi-service/pkg/service/keystore"
 	"github.com/tbd54566975/ssi-service/pkg/storage"
 )
 
 type Service struct {
-	config   config.DIDServiceConfig
-	storage  *Storage
-	resolver *didsdk.Resolver
+	config      config.DIDServiceConfig
+	storage     *Storage
+	sdkResolver *didsdk.Resolver
```

Comment:
Did this. Also had to create a LocalResolver wrapper because the SDK implementation does not accept contexts (which can arguably be part of separate issue).